cosmwasm/core.ts

```diff
@@ -93,6 +96,16 @@ const programHandler = () => {
 
     addCoreOptions(deactivateChainCmd);
 
+    const addIBCChainCmd = program
+        .command('add-ibc-chain')
+        .description('Add an IBC chain')
+        .argument('<chainName>', 'chain name to add')
+        .argument('<chainPrefix>', 'chain prefix to add')
+        .argument('<ibcPath>', 'IBC path to add')
```

Comment:
match the args match the function parameter names?
```suggestion
        .argument('<cosmosChain>', 'cosmos chain name to add')
        .argument('<addrPrefix>', 'address prefix')
        .argument('<ibcPath>', 'IBC path')
```